nav2_costmap_2d/include/nav2_costmap_2d/obstacle_layer.hpp

```diff
@@ -131,37 +140,45 @@ class ObstacleLayer : public CostmapLayer
    * @param max_x
    * @param max_y
    */
-  virtual void raytraceFreespace(const nav2_costmap_2d::Observation & clearing_observation,
-      double * min_x, double * min_y,
-      double * max_x,
-      double * max_y);
-
-  void updateRaytraceBounds(double ox, double oy, double wx, double wy, double range,
-      double * min_x, double * min_y,
-      double * max_x,
-      double * max_y);
+  virtual void raytraceFreespace(
+    const nav2_costmap_2d::Observation & clearing_observation,
+    double * min_x, double * min_y,
+    double * max_x,
+    double * max_y);
+
+  void updateRaytraceBounds(
+    double ox, double oy, double wx, double wy, double range,
+    double * min_x, double * min_y,
+    double * max_x,
+    double * max_y);
 
   std::vector<geometry_msgs::msg::Point> transformed_footprint_;
   bool footprint_clearing_enabled_;
-  void updateFootprint(double robot_x, double robot_y, double robot_yaw, double * min_x,
-      double * min_y,
-      double * max_x,
-      double * max_y);
-
-  std::string global_frame_;  ///< @brief The global frame for the costmap
-  double max_obstacle_height_;  ///< @brief Max Obstacle Height
-
-  laser_geometry::LaserProjection projector_;  ///< @brief Used to project laser scans into point clouds
-
-  std::vector<std::shared_ptr<message_filters::SubscriberBase> > observation_subscribers_;  ///< @brief Used for the observation message filters
-  std::vector<std::shared_ptr<tf2_ros::MessageFilterBase> > observation_notifiers_;  ///< @brief Used to make sure that transforms are available for each sensor
- 
-  std::vector<std::shared_ptr<nav2_costmap_2d::ObservationBuffer> > observation_buffers_;  ///< @brief Used to store observations from various sensors
-  std::vector<std::shared_ptr<nav2_costmap_2d::ObservationBuffer> > marking_buffers_;  ///< @brief Used to store observation buffers used for marking obstacles
-  std::vector<std::shared_ptr<nav2_costmap_2d::ObservationBuffer> > clearing_buffers_;  ///< @brief Used to store observation buffers used for clearing obstacles
+  void updateFootprint(
+    double robot_x, double robot_y, double robot_yaw, double * min_x,
+    double * min_y,
+    double * max_x,
+    double * max_y);
+
+  std::string global_frame_;  /// @brief The global frame for the costmap
```

Comment:
Removing the `<` from the comment is likely to screw up doxygen. It might apply the comment to the next variable instead of the one it's intended for. You should put that back, move the comment above the variable, or turn it into a regular comment instead of a doxygen comment.